File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/WorkspaceTabs/Appearance/AlignmentPage.vb
Namespace CompanionKit.Controls.WorkspaceTabs.Appearance

    Public Class AlignmentPage

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

        End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjTopRadioButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjTopRadioButton_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjTopRadioButton.CheckedChanged
            If (mobjTopRadioButton.Checked) Then
                mobjWorkspaceTabs.Alignment = TabAlignment.Top
            Else
                mobjWorkspaceTabs.Alignment = TabAlignment.Bottom
            End If
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/WorkspaceTabs/Appearance/AlignmentPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.WorkspaceTabs.Appearance
{
    public partial class AlignmentPage : UserControl
    {
        public AlignmentPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the CheckedChanged event of the objTopRadioButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjTopRadioButton_CheckedChanged(object sender, EventArgs e)
        {
            mobjWorkspaceTabs.Alignment = mobjTopRadioButton.Checked ? TabAlignment.Top : TabAlignment.Bottom;
        }


    }
}